dmoj/executors/NODEJS.py

```diff
@@ -0,0 +1,36 @@
+from dmoj.executors.script_executor import ScriptExecutor
+from dmoj.cptbox.filesystem_policies import ExactFile
+
+
+class Executor(ScriptExecutor):
+    ext = 'js'
+    command = 'node'
+    nproc = -1
+    command_paths = ['node', 'nodejs']
+    syscalls = [
```

Comment:
the linter doesn't like the formatting here

try changing `syscalls` to fit on 1 line

Reply:
Changed, and it seems like it passes `black` now